Scripts/build-cocoa.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Unless there's a reason not to, I find setting `set -euo pipefail` on the second line of every bash script helpful to avoid errors down the road (e.g. typos, silently ignored undefined variables, etc)

Reply:
Added `set -euo pipefail` for all scripts where it was missing.